src/message/messageActionSheet.js

```diff
@@ -21,6 +21,7 @@ import {
 } from '../api';
 import { showToast } from '../utils/info';
 import { doNarrow, navigateBack, startEditMessage, deleteOutboxMessage } from '../actions';
+import { navigateToEmojiPicker } from '../nav/navActions';
```

Comment:
`actions.js` contains all actions including `navActions`. So you can import `navigateToEmojiPicker ` in the previous line (
```
 import { doNarrow, navigateBack, startEditMessage, deleteOutboxMessage } from '../actions';
```)
itself.